mlrun/api/crud/model_monitoring/helpers.py

```diff
@@ -23,6 +24,16 @@
 import mlrun.common.schemas.schedule
 import mlrun.errors
 
+Seconds = typing.NewType("Seconds", int)
+Minutes = typing.NewType("Minutes", int)
+
+_SECONDS_IN_MINUTE: Seconds = 60
+_MINUTES_IN_HOUR: Minutes = 60
```

Comment:
You need to wrap the default values according to your `NewType` docstring:
```suggestion
_SECONDS_IN_MINUTE: Seconds = Seconds(60)
_MINUTES_IN_HOUR: Minutes = Minutes(60)
```

Reply:
Done - 2959854